include/xtensor/xcontainer.hpp

```diff
@@ -760,14 +760,21 @@ namespace xt
     }
 
     /**
-     * Returns the offset to the first element in the container.
+     * Returns a pointer to the underlying array serving as element storage. The pointer
```

Comment:
Should we clarify that the first element of the view is at `data() + data_offset()` ?

Reply:
Yep that's a good idea! I'll update the comment in the next commit